aten/src/ATen/native/transformers/cuda/attention.cu

```diff
@@ -585,7 +585,7 @@ std::tuple<Tensor, Tensor> _scaled_dot_product_attention_forward_cuda(
     }
 }
 
-std::tuple<Tensor, Tensor> flash_scaled_dot_product_attention(
+Tensor flash_scaled_dot_product_attention(
```

Comment:
Do we need to update this function as well? It seems to be only used by `flash_attention_helper_packed`, which is unused in this PR.

Reply:
The problem was that there was another implementation of flash_scaled_dot_prod_attention that was in nestedtransformers.cu and this one wasn't getting picked up on. So now there is one and this is it. It matches the one from nestedtransformers.cu